modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/render/DDMFormFieldValueRendererTest.java

```diff
@@ -135,7 +137,15 @@ public void testDateFieldValueRenderer() {
 		renderedValue = ddmFormFieldValueRenderer.render(
 			ddmFormFieldValue, LocaleUtil.BRAZIL);
 
-		Assert.assertEquals("22/10/2014", renderedValue);
+		String javaLocatorProviders = SystemProperties.get(
+			"java.locale.providers");
+
+		if (JavaDetector.isJDK8() || !javaLocatorProviders.equals("CLDR")) {
+			Assert.assertEquals("22/10/2014", renderedValue);
+		}
+		else {
+			Assert.assertEquals("22/10/14", renderedValue);
+		}
```

Comment:
@MrJohn1911 This is a wrong fix. You should modify production code rather than test logic as "2014" is expected based on https://liferay.atlassian.net/browse/LPS-102733